src/assets/styles/portfolio.css

```diff
@@ -0,0 +1,56 @@
+.portfolio h1 {
+  padding-top: 1rem;
+  padding-inline: 2rem;
+  padding-bottom: 0.5rem;
+  text-align: left;
+}
+
+.portfolio h2 {
+  padding-top: 1rem;
+  padding-inline: 2rem;
+  padding-bottom: 0.5rem;
+  text-align: left;
+}
+
+.portfolio h3 {
+  padding-top: 1rem;
+  padding-inline: 2rem;
+  padding-bottom: 0.5rem;
+  text-align: left;
+}
+
+.portfolio h4 {
+  padding-top: 1rem;
+  padding-inline: 2rem;
+  padding-bottom: 0.5rem;
+  text-align: left;
+}
+
+.portfolio p {
+  padding-top: 1rem;
+  padding-inline: 2rem;
+  padding-bottom: 0.5rem;
+  text-align: left;
+}
+
+.flex-grid {
+  flex-wrap: wrap;
+  flex-direction: row-reverse;
+  display: flex;
```

Comment:
Personal preference but I like to have the display mode above any display-mode-specific properties.

Reply:
Def makes sense to me. Fixed & changed my Prettier linter settings to match this preference (was missing flex-wrap & flex-direction)😄